spec/models/user_spec.rb

```diff
@@ -138,4 +140,25 @@
       expect(user.display_name).to eq('hasso01')
     end
   end
+
+  context 'with jitsi calls' do
+    let(:jitsi_calls) { FactoryBot.create_list :jitsi_call, 2 }
+
+    before do
+      user.save
+      jitsi_calls.each { |j| user.call_participants.create(jitsi_call: j, state: 'accepted', role: 'participant') }
```

Comment:
Because the tests do not depend on a specific state or role, we could use the first one from their valid lists so we do not have to change the tests if the valid lists change.

Reply:
Fixed in 2b5f129